### PR TITLE
Bat/menu example code

### PR DESCRIPTION
**Pull request by tesk9**
Adds basic partial code gen for Menu example. Doesn't support customizing entries, for instance.

### CODE REVIEW

File: styleguide-app/Debug/Control/View.elm [hunk 1]
```diff
@@ -137,7 +137,20 @@ viewSection name children =
...
-        ++ "\n\t] "
+codeFromList =
+    codeFromListWithIndentLevel 1
+
+
+codeFromListWithIndentLevel : Int -> List ( String, a ) -> String
+codeFromListWithIndentLevel indent list =
+    let
+        indents =
+            String.repeat indent "\t"
```
Comment (ap-nri) [inline styleguide-app/Debug/Control/View.elm:148]: This leaves us with invalid elm if we want to copy, but that's ok right?

**Review by ap-nri — APPROVED** [review]
@NoRedInkBot r+

**Comment by ap-nri** [comment]
merged with a note about tabs being invalid Elm because that's just not a blocker

File: styleguide-app/Debug/Control/View.elm (hunk 1 above)
Comment (tesk9) [reply styleguide-app/Debug/Control/View.elm:148]: Thanks for this note! I have the tabs replaced in the Ellie version, but I think you're right that I should replace them in the rendered examples too.

**Pull request merged**